profile/README.md

```diff
@@ -0,0 +1,5 @@
+* [🫶  Learn how to get involved](https://hepsoftwarefoundation.org/get_involved.html)
+* [🎓 Browse the GitHub organization of our training WG](https://github.com/hsf-training/)
+* [🐛 Find good first issues on github to start contributing](https://github.com/issues?q=is%3Aissue+is%3Aopen+archived%3Afalse+sort%3Aupdated-desc+label%3A%22good+first+issue%22+org%3Ahsf)
+* [🦜 Stay up to date with our twitter feed](https://twitter.com/hepsoftfound)
+[🦉 Browse our calendar of meetings](https://indico.cern.ch/category/5816/)
```

Comment:
Sorry, I forgot the bullet.
```suggestion
* [🦉 Browse our calendar of meetings](https://indico.cern.ch/category/5816/)
```

Reply:
Or https://indico.cern.ch/category/5816/calendar? The website calendar includes a bit more 'noise' from other organizations. Either way.